sdk/translation/azure-ai-translation-text/azure/ai/translation/text/_patch.py

```diff
@@ -56,6 +56,29 @@ def on_request(self, request: PipelineRequest) -> None:
         request.http_request.headers["Ocp-Apim-Subscription-Key"] = self.credential.key
         request.http_request.headers["Ocp-Apim-Subscription-Region"] = self.credential.region
 
+class TranslatorAADCredential:
+    """Credential for Translator Service when using AAD authentication."""
+
+    def __init__(self, tokenCredential: TokenCredential, resourceId: str, region: str) -> None:
+        self.tokenCredential = tokenCredential
+        self.resourceId = resourceId
+        self.region = region
+
+class TranslatorAADAuthenticationPolicy(BearerTokenCredentialPolicy):
```

Comment:
Can you add docs for `translatorCredential`

Reply:
Sure